patches/common/chromium/dcheck.patch

```diff
@@ -224,3 +224,26 @@ index 674b0e9a909c..a1bff6e40f56 100644
      FreeData(format, handle);
    }
  }
+diff --git a/base/memory/weak_ptr_unittest.cc b/base/memory/weak_ptr_unittest.cc
+index d223bd2bf6d8..825bbbd85aac 100644
+--- a/base/memory/weak_ptr_unittest.cc
++++ b/base/memory/weak_ptr_unittest.cc
+@@ -565,7 +565,9 @@ TEST(WeakPtrTest, NonOwnerThreadCanDeleteWeakPtr) {
+   background.Start();
+   background.DeleteArrow(arrow);
+ }
+-
++/*Disabling the WeakPtrDeathTest tests until WeakReference::Flag::IsValid() 
++DCHECK is fixed*/
++/*
+ TEST(WeakPtrDeathTest, WeakPtrCopyDoesNotChangeThreadBinding) {
```

Comment:
Can you rename the test desc with `DISABLED_` similar to other disabled tests instead of commenting it out. Thanks!